src/test/java/com/synopsys/integration/detect/battery/BatteryTest.java

```diff
@@ -59,6 +59,7 @@
 import freemarker.template.TemplateException;
 
 public final class BatteryTest {
+    private static final String ENVIRONMENT_VARIABLE_BATTERY_TESTS_PATH = "BATTERY_TESTS_PATH";
```

Comment:
Changes in this class are the real changes to fix the issue with testAll not executing the battery tests.